resty

```diff
@@ -168,6 +170,17 @@ for my $var (sort keys %ENV) {
     $env_list .= "env $var;\n";
 }
 
+my $main_include_directive = '';
+my $http_include_directive = '';
+
+if ($main_include) {
+    $main_include_directive = "include $main_include;"
```

Comment:
Well, it would limit it somewhat, since the include directive can take wildcards. That's implicitly supported now (by quoting the command line argument), but obviously wouldn't work if we require it to be a file. Also, I think it will be readily apparent if the file doesn't exist as nginx will fail immediately, so it boils down whether it's preferable to have the `resty` command error out instead of the subsequent nginx invocation. Given that it's more restrictive and little is gained I would vote for not doing it, but it's your call :)